NuRadioReco/modules/triggerTimeAdjuster.py

```diff
@@ -177,7 +197,7 @@ def run(self, event, station, detector, mode='sim_to_data'):
             for pre_trigger_time in pre_trigger_times:
                 if pre_trigger_time is not None:
                     for channel in station.iter_channels():
-                        channel.set_trace_start_time(channel.get_trace_start_time()-pre_trigger_time[channel.get_id()])
+                        channel.set_trace_start_time(channel.get_trace_start_time() - pre_trigger_time[channel.get_id()])
```

Comment:
See also the discussion in #526 - @cg-laser wanted the traces from simulation to all start at the same time, and store the pre_trigger_times in the Trigger object, so in an analysis one has to run the triggerTimeAdjuster with `mode="data_to_sim"` to get the correct (adjusted) trace start times, analogously to how the `cableDelayAdder` works. 

I actually argued at the time for your implementation, but had no supporters - happy to reopen the argument, but if we stick with your implementation we should remove the "data_to_sim" option.